osu.Game/Overlays/BeatmapListing/BeatmapListingFilterControl.cs

```diff
@@ -24,6 +22,8 @@ public class BeatmapListingFilterControl : CompositeDrawable
     {
         public Action<List<BeatmapSetInfo>> SearchFinished;
         public Action SearchStarted;
+        /// <summary> List of currently displayed beatmap entries </summary>
```

Comment:
Again, mostly unnecessary, but if you're going to add xmldoc, please do it consistently and don't single-line it like this.
```suggestion
        /// <summary>
        /// List of currently displayed beatmap entries.
        /// </summary>
```

Reply:
To removal